magiccube/cube.py

```diff
@@ -315,6 +315,42 @@ def reverse_history(self, to_str=False):
 
         return reverse
 
+    @property
+    def get_facelet_colors_in_kociemba_order(self) -> str:
+        faces = [
+            ''.join(
+                [
+                    fc.name
+                    for fc in self.get_face_flat(
+                            Face.create(f),
+                    )
+                ],
+            )
+            for f in ['U', 'R', 'F', 'D', 'L', 'B']
+        ]
+
+        return ''.join(faces)
+
+    @property
+    def as_kociemba_facelets(self) -> str:
```

Comment:
```suggestion
    def get_kociemba_facelet_positions(self) -> str:
        """Return the string representation of the cube facelet positions in Kociemba format (ex: UUUUUUUUURRRRRRRRRFFFFFFFFFDDDDDDDDDLLLLLLLLLBBBBBBBBB)."""
```
Leaving here the suggestion to rename the method to use the same pattern as other methods and have a docstring to help other API users.

Reply:
Suggestions applied.